packages/cactus-test-tooling/src/main/typescript/quorum/quorum-test-ledger.ts

```diff
@@ -18,7 +18,7 @@ export interface IQuorumTestLedgerConstructorOptions {
 }
 
 export const QUORUM_TEST_LEDGER_DEFAULT_OPTIONS = Object.freeze({
-  containerImageVersion: "2021-01-08-7a055c3",
+  containerImageVersion: "latest",
```

Comment:
@jagpreetsinghsasan Please specify a tag that's designed to not change over time: This is to ensure that the CI/tests consistently succeed or fail while being executed with a given revision (git sha) of the code.